php/www/event.php

```diff
@@ -0,0 +1,49 @@
+<?php
+require_once("./lib/connect-db.php");
+require_once("./components/header/index.php");
+require_once("./components/sidebar/index.php");
+require_once("./components/article-card/index.php");
+require_once("./components/content-card/index.php");
+?>
+
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <title>学内掲示板</title>
+    <?php require_once('./lib/bootstrap.php'); ?>
+    <link rel="stylesheet" href="/style/main.css">
+</head>
+
+<body>
+    <div class="d-flex">
+        
+        <?php 
+            $sidebar = new Sidebar();
+            $sidebar->render();
+        ?>
+
+        <!-- メインコンテンツ -->
+        <div class="content w-100 ms-4">
```

Comment:
```suggestion
        <div class="content w-100 ms-md-4>
```

モバイル時に右端に余白ができるので表示しないように切り替えてください。

<img width="645" alt="image" src="https://github.com/user-attachments/assets/2277de5b-6141-4733-b02b-8b4ad404d951" />

Reply:
他の画面も同じく